### ISSUE TITLE
proposal: use GitHub pages website

### ISSUE THREAD

**Issue by plata** [PhoenicisOrg/phoenicis#1344]
I experienced that phoenicis.org is not really fast if your internet connection is a bit slow. As I wanted to play around with Jekyll anyways, I tried to implement something similar to phoenicis.org in Jekyll. The result is: https://plata.github.io/phoenicis-website-jekyll/.

I propose to use this as our website:
* it's easier to maintain (just GitHub pages)
* no additional servers/accounts required
* easy to contribute and update (simply PR)
* faster than phoenicis.org

**Comment by madoar** [comment]
The navigation bar at the top is missing an opaque background, which becomes a problem during scrolling.

I like the look, but I'm quite unsure how we would maintain the website.
The advantage of using angular (js or 2+) with a database in the background is, that it enables us to add a web interface, over which the whole content can be managed, similar to a content management system.
Therefore it doesn't require any coding knowledge to add new news articles or other kinds of content, we add later on to the website.

**Comment by qparis** [comment]
Agreed. Also, this version was a proof of concept and cat clearly be optimized with newer technology (with prerendering for example)

What about the list of available applications? Is it dynamic?

**Comment by plata** [comment]
Yes, it's this: https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js

**Comment by qparis** [comment]
Seriously, I strongly disagree with the fact that generating static pages is more maintenable. 

You can do a *lot* more with the dynamic approach, including of course compressing the miniature.

So far, you are comparing static pages with a SPA made with an old piece of technology (AngularJS), so it it absolutely not surprising to see that you are performing better. 

However, in a long term thinking, I strongly encourage to use a real website (AngularJS is challengeable though) that will open the door to many really useful services (for example contributing directly from the website, an admin interface for the news, some click & install features, the list of wine engines, maybe the ability to ask for custom builds and follow the compilation progress, etc, etc, ... the list is infinite)

For a first step, I suggest we keep Phoenicis-website, but I think we need to get rid of Angular and use server side templating instead. This shall improve the performances up to the same level you are experiencing right now with your prototype

**Comment by plata** [comment]
Fine for me as long as you can find the time to maintain it because I definitely won't.

**Comment by qparis** [comment]
Can you provide me the source of your website so that I can use it as a base for an improved version of phoenicis website?

**Comment by plata** [comment]
Sure. It's GitHub pages, so all the sources are here: https://github.com/plata/phoenicis-website-jekyll

**Comment by plata** [comment]
Depending on the time it will take to implement the new website: we could consider using the GitHub pages website just until the "real" one is done.

**Comment by madoar** [comment]
@plata I have no problem accessing phoenicis.org

**Comment by plata** [comment]
Yes, it's back. Have you ever tried using it on a mobile?

**Comment by qparis** [comment]
@plata Weird. Any issue yet?

**Comment by madoar** [comment]
I've checked it on my android smartphone.
The website is accessible, but it is not adjusted for smartphone usage.

**Comment by plata** [comment]
@madoar can you go to anywhere except the main page?

**Comment by plata** [comment]
We will enhance the existing playonlinux.com.
